mcs/class/corlib/wasm_corlib.dll.sources

```diff
@@ -0,0 +1,5 @@
+#include corlib.dll.sources
+
+corert/ThreadPoolBoundHandle.platformnotsupported.cs
+corert/PreAllocatedOverlapped.platformnotsupported.cs
```

Comment:
hmm, I think it might be easier to add these to corlib.dll.sources instead and exclude them just in win32_net_4_x_corlib.dll.exclude.sources and winaot_corlib.dll.exclude.sources. That way we shouldn't have to add .sources for all the profiles.

Reply:
Good idea.  I really didn't like the addition of these one million additional .sources files.